### ISSUE TITLE
Turn AWS region parameter into an enum?

### ISSUE THREAD

**Issue by heiko-braun** [apache/camel-k#3289]
We see a number of errors when creating AWS kamelets because people provide a wrong value the region parameter.

It can be prevented if we would turn region into an enum type, rather a string.

```
Caused by: org.apache.camel.RuntimeCamelException: software.amazon.awssdk.core.exception.SdkClientException: Configured region (us-east-1.console.aws.amazon.com%2Fec2%2Fv2%2Fhome%3Fregion%3Dus-east-1%23Instances%3A) and tags ([]) resulted in an invalid URI: [https://kinesis.us-east-1.console.aws.amazon.com%2Fec2%2Fv2%2Fhome%3Fregion%3Dus-east-1%23Instances%3A.amazonaws.com](https://kinesis.us-east-1.console.aws.amazon.com%2Fec2%2Fv2%2Fhome%3Fregion%3Dus-east-1%23instances:.amazonaws.com/). This is usually caused by an invalid region configuration. Valid regions: [af-south-1, ap-east-1, ap-northeast-1, ap-northeast-2, ap-northeast-3, ap-south-1, ap-southeast-1, ap-southeast-2, ap-southeast-3, ca-central-1, eu-central-1, eu-north-1, eu-south-1, eu-west-1, eu-west-2, eu-west-3, fips-us-east-1, fips-us-east-2, fips-us-west-1, fips-us-west-2, me-south-1, sa-east-1, us-east-1, us-east-2, us-west-1, us-west-2, cn-north-1, cn-northwest-1, us-gov-east-1, us-gov-west-1, us-iso-east-1, us-iso-west-1, us-isob-east-1]
at org.apache.camel.RuntimeCamelException.wrapRuntimeCamelException(RuntimeCamelException.java:51)
at org.apache.camel.support.ChildServiceSupport.start(ChildServiceSupport.java:67)
at org.apache.camel.support.service.ServiceHelper.startService(ServiceHelper.java:113)
at org.apache.camel.support.service.ServiceHelper.startService(ServiceHelper.java:130)
```

**Comment by oscerd** [comment]
It will be closed once we switch to catalog 0.9.0

**Comment by tadayosi** [comment]
Already switched to Kamelet Catalog 0.9.0. Closing.
